docs/days/day-6.md

```diff
@@ -127,6 +127,10 @@ In the MVC model, the Model defines all the business logic, and the Controller o
 As the code returns a collection of jobs, let’s move the code to the repository, that is part of model layer.
 For that we will need to create a custom `repository` class for Job entity and to add the query to that class.
 
+At this point, you may have a question - what is this Repository class anyway? 
+Repository is a pattern, a common solution to the well know problem. It has been around for quite some time and was popularizer around 90's by people like Martin Forwler and Eric Evans. If you try to describe it briefly it can get quite complex. For now think of it as just another layer of abstraction
```

Comment:
Fowler, not Forwler

Reply:
popularized, not popularizer

Reply:
one of these days I will get enough sleep